exercises.json

```
@@ -0,0 +1,10 @@
+{
+  "exercises": [
+    {
+      "name": "strings1",
+      "path": "exercises/strings/strings1.js",
+      "hint": "Add your hint here",
```

Can hint be an array? Wherein we can provide hints and keep an index count of which hint we are on. We can stick to one hint for now, but I think making it an array would be better?
```suggestion
      "hint": ["Add your hint here"],
      "currentHintIndex" : 0, 
```

---

Ah that's a great idea. The `hint` command should have an `--all` option to display all hints!